driver/razermousemat_driver.c

```diff
@@ -40,12 +40,14 @@ MODULE_DESCRIPTION(DRIVER_DESC);
 MODULE_VERSION(DRIVER_VERSION);
 MODULE_LICENSE(DRIVER_LICENSE);
 
+static unsigned char saved_brightness = 0;
 
 /**
  * Send report to the mousemat
  */
 int razer_get_report(struct usb_device *usb_dev, struct razer_report *request_report, struct razer_report *response_report)
 {
+    // TODO: Rename WAIT_MIN_US from FIREFLY
```

Comment:
solve that todo please, should be a simple find&replace

Reply:
Done